scripts/get-dep-lib.sh

```diff
@@ -103,7 +103,7 @@ fi
 
 # FIX FOR ANDROID,
 if [ "$PLATFORM" == "singleAndroid" ] ; then
-	CURRENT_OPENSSL="OpenSSL_1_0_2o"
+	CURRENT_OPENSSL="OpenSSL_1_1_1c"
```

Comment:
Can't we harmonize the openssl version now, and remove this kludge?

Reply:
Fully agree that all the scripting around openssl version is a mess. But, I believe its beyond the scope op this PR, as its Android only here, and I think a real solution would be over all build platforms and flavors. SO, NAK here, but ACK in general.

Reply:
I was guessing that it should be a easy upgrade from 1.1.0h to 1.1.1c to harmonize everything.

Reply:
if there is one thing that I have learned about OpenSSL... "easy" is never part of the answer :-/